estimation/basecamp/moments_calculation.py

```diff
@@ -44,7 +45,7 @@ def get_moments(data, is_store = False):
 
     # Save mean and standard deviation of wages for each period
     # to Wage Distribution section of the moments dictionary
-    for period in range(30):  ## TO DO: Remove hard coded number
+    for period in range(30):  # TO DO: Remove hard coded number
```

Comment:
Please endogenize or put as a module-wide variable at beginning of the module

Reply:
Setting to beginning of module, will fully endogenize at some future point in time